### ISSUE TITLE
Extension author text in list view does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51984]
**Environment Details**:
VSCode Version : 1.24.1
OS Version : Win10
**Additioanl Details:**
ColorContrast,MAS1.4.3
**Repro Steps:**
1. Launch VS Code.
2. Navigate to View(Alt+V) menu option and select "Extensions" from dropdown.
3. Open color contrast analyzer and check the ratio.

**Actual:**
Color Contrast ratio is less than 4.5:1 for some content present on the screen.
Same issue observed on all other color themes.
**Expected:**
Color Contrast ratio should be greater than 4.5:1.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not able to see the content which is written on the screen.
**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
Does this issue occur when all extensions are disabled?: Yes
**Attachment Reference**
![accessibility a11y_vscode_viewextensions_colorcontrast](https://user-images.githubusercontent.com/39204909/41458869-72dd9696-70a5-11e8-8535-704b9c23e987.png)
![accessibility a11y_vscode_viewextensions_colorcontrast1](https://user-images.githubusercontent.com/39204909/41458870-73124d46-70a5-11e8-9747-1d35a10c9e0d.png)
